ai_worker/.env.example

```
@@ -0,0 +1,76 @@
+# LEH AI Worker - Environment Variables Example
+# Copy this file to .env and fill in your values
+
+# ============================================
+# AWS Configuration
+# ============================================
+
+# S3 Configuration
+AWS_REGION=ap-northeast-2
+S3_EVIDENCE_BUCKET=leh-evidence-bucket-{case_id}
+S3_DOWNLOAD_TMP_PATH=/tmp
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The S3_DOWNLOAD_TMP_PATH key should go before the S3_EVIDENCE_BUCKET key

(UnorderedKey)